code/game/dna/genes/vg_powers.dm

```diff
@@ -240,7 +240,10 @@ Obviously, requires DNA2.
 
 /obj/effect/proc_holder/spell/targeted/remotetalk/cast(list/targets)
 	if(!ishuman(usr))	return
-	var/say = strip_html(input("What do you wish to say"))
+	var/say = input("What do you wish to say") as text|null
+	if(say == null || length(say) == 0)
```

Comment:
Because of how BYOND is weird and treats null, 0, and "" all the same, this might work simply as `if(!say)`.